blog/2019-04-25-pulsar-debezium-tutorial.adoc

```diff
@@ -0,0 +1,242 @@
+= Tutorial for Pulsar Debezium Connecttor
+jiazhai, StreamNative
+:awestruct-tags: [ discussion, examples  ]
+:awestruct-layout: blog-post
+
+https://debezium.io[Debezium] is an open source project for change data capture (CDC). It is built on Kafka Connect and supports multiple databases, such as MySQL, MongoDB, PostgreSQL, Oracle, and SQL Server. 
+http://pulsar.apache.org[Apache Pulsar] provides a http://pulsar.apache.org/docs/en/2.3.0/io-cdc-debezium[source connector] over Kafka connect, so you can leverage Debezium to stream changes from your databases. 
+This tutorial walks you through using Pulsar source connector to store MySQL data changes. 
+
+## Tutorial Steps
+This tutorial is similar to https://debezium.io/docs/tutorial[debezium tutorial], except that storage of event streams is changed from Kafka to Pulsar.
+It mainly includes six steps: 
+1. Start a MySQL server;
+2. Start standalone Pulsar service;
+3. Start Pulsar connector. Pulsar connector reads database changes existing in MySQL server;
+4. Subscribe Pulsar topics to monitor MySQL changes;
+5. Make changes in MySQL server, and verify that changes are recorded in Pulsar topics immediately;
+6. Clean up.
+
+### Step 1: Start a MySQL server
+Start a MySQL server that contains a database example, from which Debezium captures changes. Open a new terminal to start a new container that runs a MySQL database server pre-configured with a database named inventory:
+
+[source,bash]
+----
+docker run --rm --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw debezium/example-mysql:0.8
+----
+
+The following information is displayed:
+
+[source,bash]
+----
+2019-03-25T14:12:41.178325Z 0 [Note] Event Scheduler: Loaded 0 events
+2019-03-25T14:12:41.178670Z 0 [Note] mysqld: ready for connections.
+Version: '5.7.25-log'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
+----
+
+### Step 2: Start standalone Pulsar service
+Start Pulsar service locally in standalone mode.
+Debezium connector is introduced in Pulsar 2.3.0. 
```

Comment:
Perhaps "Support for running Debezium connectors in Pulsar is introduced in Pulsar 2.3.0"?